.editorconfig

```diff
@@ -1,7 +1,10 @@
 root = true
 
 [*]
+end_of_line = lf
```

Comment:
Was this meant to be in this PR? I saw the discussion in the #v3-markathon channel, and I'm not sure there was a decision made on this. I'd prefer this to be in a separate PR (if at all :)).

Reply:
I can pull it out and open a new PR for discussion.  It seems clear to me that our line endings for `*.md` files at LEAST should be consistent across the whole repository.  But discussion for another place.

Reply:
> I can pull it out and open a new PR for discussion. It seems clear to me that our line endings for *.md files at LEAST should be consistent across the whole repository. But discussion for another place.

Seems like a plan!